src/components/custom/_signup-default.tsx

```diff
@@ -136,15 +136,15 @@ const SignupDefault = ({
                 </SocialButton>
             </SocialWrapper>
             <LoginText>
-                {localize('Already have an account?')}
+                <Localize translate_text="_t_Already have an account?_t_" />
                 <LoginLink id="dm-default-login-button" onClick={handleLogin}>
                     {' '}
```

Comment:
Can we remove it? Looks like useless line

Reply:
It is to keep a space between "Already have an account?" and "Log in." if removed and add space in the translate text, may affect the translation.